### ISSUE TITLE
qrcode does not work with core

### ISSUE THREAD

**Issue by eisenherzz** [cworreschk/grav-plugin-qrcode#4]
Installing latest Version of qrcode with admin plugin everything works as expected.
Without admin plugin (see [test.eisenherzz.de](test.eisenherzz.de))
```
InvalidArgumentException thrown with message "Identifier "user" is not defined."

Stacktrace:
#14 InvalidArgumentException in /www/htdocs/w0071ea7/test/vendor/pimple/pimple/src/Pimple/Container.php:96
#13 Pimple\Container:offsetGet in /www/htdocs/w0071ea7/test/user/plugins/qrcode/qrcode.php:30
#12 Grav\Plugin\QrCodePlugin:initializeLocale in /www/htdocs/w0071ea7/test/user/plugins/qrcode/qrcode.php:65
#11 Grav\Plugin\QrCodePlugin:onPluginsInitialized in /www/htdocs/w0071ea7/test/vendor/symfony/event-dispatcher/EventDispatcher.php:184
#10 call_user_func in /www/htdocs/w0071ea7/test/vendor/symfony/event-dispatcher/EventDispatcher.php:184
#9 Symfony\Component\EventDispatcher\EventDispatcher:doDispatch in /www/htdocs/w0071ea7/test/vendor/symfony/event-dispatcher/EventDispatcher.php:46
#8 Symfony\Component\EventDispatcher\EventDispatcher:dispatch in /www/htdocs/w0071ea7/test/vendor/rockettheme/toolbox/Event/src/EventDispatcher.php:23
#7 RocketTheme\Toolbox\Event\EventDispatcher:dispatch in /www/htdocs/w0071ea7/test/system/src/Grav/Common/Grav.php:277
#6 Grav\Common\Grav:fireEvent in /www/htdocs/w0071ea7/test/system/src/Grav/Common/Processors/PluginsProcessor.php:19
#5 Grav\Common\Processors\PluginsProcessor:process in /www/htdocs/w0071ea7/test/system/src/Grav/Common/Grav.php:126
#4 Grav\Common\Grav:Grav\Common\{closure} in /www/htdocs/w0071ea7/test/system/src/Grav/Common/Grav.php:368
#3 Grav\Common\Grav:Grav\Common\{closure} in /www/htdocs/w0071ea7/test/system/src/Grav/Common/Grav.php:344
#2 call_user_func_array in /www/htdocs/w0071ea7/test/system/src/Grav/Common/Grav.php:344
#1 Grav\Common\Grav:__call in /www/htdocs/w0071ea7/test/system/src/Grav/Common/Grav.php:127
#0 Grav\Common\Grav:process in /www/htdocs/w0071ea7/test/index.php:52
```
Older version works without admin.

**Comment by cworreschk** [comment]
Thank you for your bug report. It's fixed with version 1.0.3

**Comment by eisenherzz** [comment]
Works fine now.
